components/locid/src/langid.rs

```diff
@@ -60,6 +60,7 @@ use alloc::string::ToString;
 ///
 /// [`Unicode BCP47 Language Identifier`]: https://unicode.org/reports/tr35/tr35.html#Unicode_language_identifier
 #[derive(Default, PartialEq, Eq, Clone, Hash, PartialOrd, Ord)]
+#[allow(clippy::exhaustive_structs)] // This struct is stable (and invoked by a macro)
```

Comment:
Thought: This one is OK. We have a lot of code that relies on the stability of this.

Reply:
Yeah I think this one is part of why I wasn't quite sure